posts/2020-01-28-conf-lineup.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "The 2020 Rust Event Lineup"
+author: Rust Community
+description: "Welcome to 2020; We are excited about the Rust conferences coming up; join us at one near you!"
+---
+
+
+A new decade has started, and we are excited about the Rust conferences coming up. Each conference is an opportunity to learn about Rust, share your knowledge, and to have a good time with your fellow rustacians. Read on to learn more about the events we know about so far.
+
+---
+
+**FOSDEM** <br> February 2nd, 2020 
+---
+
+[FOSDEM][fosdem site] stands for the Free and Open Source Developers European Meeting. At this event software developers around the world will meet up, share ideas and collaborate. FOSDOM will be hosting a [Rust devroom][fosdem agenda] workshop that aims to present the features and possibilities offered by Rust, as well as some of the many exciting tools and projects in its ecosystem.
+
+[fosdem site]: https://fosdem.org/2020/
+[fosdem agenda]: https://fosdem.org/2020/schedule/track/rust/
+
+###### Located in *Brussels, Belgium*
+
+
+---
+
+**Rust LATAM**<br>May 22nd-23rd, 2020
+---
+### 
+
+Where Rust meets Latin America! [Rust Latam][latam-site] is Latin America's leading event for and by the Rust community. Two days of interactive sessions, hands-on activities and engaging talks to bring the community together. Schedule to be announced [at this link][latam-agenda].
+
+[latam-site]: https://rustlatam.org/
+[latam-agenda]: https://rustlatam.org/#schedule
+
+###### Located in *Mexico City, Mexico*
+---
+
+**RustConf**<br>August 20th-21st, 2020
+---
+
+The official [RustConf][conf-site] will be taking place in Portland, Oregan, USA. Last years conference was amazing, and we are excited to see what happens next. See the [website][conf-site], and [Twitter][conf-twitter] for updates as the event date approaches!
+
+[conf-site]: https://rustconf.com/
+[conf-twitter]: https://twitter.com/rustconf
+
+###### Located in *Oregan, USA*
+
+---
+
+**RustLab**<br>October 16th-17th, 2020
+---
+
+[RustLab 2020][lab-site] is a 2 days conference with talks and workshops. The date is set, but the talks are still being planned. We expect to learn more details as we get closer to the date of the conference.
+
+[lab-site]: https://www.rustlab.it
+
+###### Located in *Florence, Italy*
+
+---
```

Comment:
**Rust/GNOME hackfest Spring 2020**<br>Apr, 29th - May, 3rd
---

The goal of these [hackfests][hack-link] is to improve the interactions between Rust and the GNOME libraries. In this hackfest we want to improve the interoperability between Rust and GNOME, improve support of GNOME libraries in Rust and explore solutions to create GObject APIs from Rust. 

[hack-link]: https://wiki.gnome.org/Hackfests/Rust2020

###### Located in **Montréal, Quebec**